.codeclimate.yml

```diff
@@ -1,6 +1,6 @@
 engines:
-  rubocop:
+  eslint:
     enabled: true
 ratings:
   paths:
-  - "**.rb"
+  - "**eslint"
```

Comment:
If it's just the one file, I think it'd be better to put it explicitly here: `- bin/eslint`